### ISSUE TITLE
Ai and aligns first 3 steps

### ISSUE THREAD

**Issue by islam2hamy** [calcitem/Sanmill#283]
The AI let me aligns first three pieces every time 
1.     a1    d6
 2.     d1    d2
 3.     g1xd2    f4

Is it a bug?

**Comment by calcitem** [comment]
Yes, it is indeed an obvious bug. Thanks for the feedback. I will look into it to see how to fix it.

**Comment by calcitem** [comment]
After some research, we found that White had formed a Mill first along the side, while problematic, does not usually affect the larger game.

However, we plan to look further into this opening sequence, so thank you for letting us know.

**Comment by calcitem** [comment]
If you turn off the "Draw on the human experience" in the Preferences, Sanmill will block the mill.
